tracer/test/snapshots/Security.AspNetCore5.enableIast=True.path =_Iast_AllVulnerabilitiesCookie.verified.txt

```diff
@@ -44,6 +44,69 @@
       "evidence": {
         "value": "AllVulnerabilitiesCookieKey"
       }
+    },
+    {
+      "type": "NO_SAMESITE_COOKIE",
+      "hash": -1837181716,
+      "evidence": {
+        "value": "LongCookie.abcdefghijklmnopqrstuvwxyz0123456789.0"
+      }
+    },
```

Comment:
Is this all correct? 🤔 I'm not sure exactly what the "grouping" is you're talking about... Shouldn't these be filtered (ignored) because they're long?

Reply:
No, all the "filtered" cookies have the same hash for the same vulnerability. Maybe "filter" is not the best word to express what is being done and "group" could be better, but...

Reply:
Yeah, filter for something we're not filtering is a terrible name 😂 Ah well